docs/content/docs/Technical/api.md

```diff
@@ -190,4 +190,48 @@ requests.post(
     headers={"Authorization": f"Bearer {self.api_token}"},
     json={"value": ["Civilian-military interaction", "Protest"], "message": "This is a comment."},
 )
-```
\ No newline at end of file
+```
+
+
+### Create an incident
+`POST /api/v2/incidents/new` creates a new incident. It has two required parameters:
+- `description`, the incident's description. `description` should be a string of at least 8 characters.
+- `sensitive`, a string array of the incident's sensitivity. That should be either `["Not Sensitive"]`, or any combination of the values `["Graphic Violence", "Deceptive or Misleading", "Personal Information Visible"]`.
```

Comment:
Can we link to the other place in the docs where we explain this attribute (if that exists)?